src/scheduler.rs

```diff
@@ -30,6 +32,54 @@ pub fn run_scheduler(
     tokio::spawn(async move { scheduler_loop(config_store, checker, producer, shutdown).await })
 }
 
+fn record_result_metrics(result: &CheckResult) {
+    // Record metrics
+    let CheckResult {
+        status,
+        scheduled_check_time,
+        actual_check_time,
+        duration,
+        status_reason,
+        ..
+    } = result;
+
+    // Record duration of check
+    if let Some(duration) = duration {
+        Metric::timing("http_checker.duration", duration.to_std().unwrap()).send();
```

Comment:
Should we have tags with the result on these (and all metrics)?

Reply:
good idea 👍